File: PHP/UberBlack.php
<?php 
class UberBlack extends Car {
    public $typeCarAccepted = array(array());
    public $seatMaterials = array();

    public function __construct($license, $driver, $typeCarAccepted, $seatMaterials) {
        parent::__construct($license,$driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->$seatMaterials = $seatMaterials;
      }

}
?>

File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

import CursoPOOUber.Java.Account;
import CursoPOOUber.Java.Car;

public class UberBlack extends Car {
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String license, Account driver,
    Map<String, Map<String,Integer>> typeCarAccepted,
    ArrayList<String> seatsMaterial){
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}
